apis/resourcecontrollerv2/v1alpha1/resourcekey_types.go

```diff
@@ -84,6 +80,9 @@ type ResourceKeyObservation struct {
 	// The short ID of the resource group.
 	ResourceGroupID string `json:"resourceGroupId,omitempty"`
 
+	// The CRN of resource instance or alias associated to the key.
+	SourceCrn string `json:"sourceCrn,omitempty"`
+
```

Comment:
Also not sure if we want this or not, but easy to fix.

Reply:
yes, I think we should have that, probably forgot to add